packages/cloud-cognitive/src/components/TagSet/TagSet.js

```diff
@@ -304,6 +297,15 @@ TagSet.propTypes = {
    * label for the overflow show all tags link. **Note: Required if more than 10 tags**
    */
   showAllTagsLabel: string_required_if_more_than_10_tags,
+  /**
+   * An array of Carbon objects containing Carbon Tag properties plus 'label'. NOTE: 'filter' is not ignored.
+   */
+  tags: PropTypes.arrayOf(
+    PropTypes.shape({
+      ...prepareProps(Tag.propTypes, 'filter'),
```

Comment:
Why are we removing 'filter' from the shape? I saw the comment about how 'filter' is NOT ignored -- if it's not ignored, are people meant to be able to supply it? And if they are meant to supply it, why not include it in the shape? Removing it in the shape doesn't stop them supplying it, of course, but it does remove the type checking from it, and I'm not sure why we'd want to do that. If this is doing something important, perhaps add some clearer comment explanation as to why we're excluding it from the shape and then using it anyway.

```suggestion
      ...Tag.propTypes,
```

Reply:
Ooops. one too many instances of 'not', namely the one the NOTE is about.